src/schema/rules/sidecars/eyetrack.yaml

```
@@ -0,0 +1,48 @@
+#
+# Groups of related metadata fields
+#
+# Assumptions: never need disjunction of selectors
+# Assumptions: top-to-bottom overrides is sufficient logic
+---
+EyeTrackingHardware:
+  selectors:
+    - datatype == "eyetrack"
+    - suffix == "eyetrack"
+  fields:
+    Manufacturer: recommended
+    ManufacturersModelName: recommended
+    SoftwareVersions: recommended
+    DeviceSerialNumber: recommended
+
+EyeTrackingRequired:
+  selectors:
+    - datatype == "eyetrack"
+    - suffix == "eyetrack"
+  fields:
+    EnvironmentCoordinates: required
+    SamplingFrequency: required
```

```suggestion
    SamplingFrequency_eyetrack: required
```

By allowing setting `n/a` here, then `_eyetrack.tsv.gz` could also contain non-uniformly sampled data (thinking of encoding the *events*, which, at least for EyeLink, only sample during fixations and saccades.

---

I think we are not opening this can of worms for now. I'll mark as resolved.